blocks/product-details/product-details.js

```diff
@@ -260,6 +249,7 @@ export default async function decorate(block) {
               });
             },
           },
+          useACDL: true,
```

Comment:
Does this mean that ACDL is on opt-in only basis? I thought it was implemented OOTB in all dropins. Is this a pattern we should follow for all other dropins?

Reply:
No, I don't think this should be a pattern followed in the other dropins, and eventually I might even remove it from the PDP dropin.
I had to add this because I have no control of how/when Bulk will update the PDP version and they are already handling the data collection on the product details block directly and I didn't want to interfere with that.